### ISSUE TITLE
🐛 Bug Report: Functions(client).list return false instead of empty dictionary as response when  no variables are set

### ISSUE THREAD

**Issue by Pastajello** [appwrite/sdk-for-dart#22]
### 👟 Reproduction steps

When a function has no set variables it is displayed correctly in web panel as 
![image](https://user-images.githubusercontent.com/28345098/180037473-7fda48d7-0f87-4bec-b61e-51f89c8fe664.png)
but what is returned by Functions(client).list is:
![image](https://user-images.githubusercontent.com/28345098/180037552-2886c17d-3568-4ad2-bca6-4dbaf698f145.png)
thus producing an exception in mapping:
![image](https://user-images.githubusercontent.com/28345098/180037655-854cc690-bd41-487f-854c-7d4c8801a372.png)

### 👍 Expected behavior

Empty vars should return empty dictionary, method works correctly

### 👎 Actual Behavior

empty vars return false, method throws an exception

### 🎲 Appwrite version

Version 0.15.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
> Let's move it there, also I'll test it soon and let you know.

Moved to sdk-for-dart. Sorry I moved it into the wrong repo.

@Pastajello, @lohanidamodar, I checked migrated functions and newly created functions via the Appwrite Web Console and `vars` is returned as `{}` for both cases. :monocle_face: How did you create your function?

**Comment by Pastajello** [comment]
I created it with web panel O_O. But I must admin I had a lot of problems with this installation as I changed VPS. Had problems with runtime warmpus with custom docker compose(my .env), so I installed a 'default' appwrite with installation command from main page then changed the .env and recreated it with docker compose up. 
Maybe this somehow corrupted my installation ?_? 
But at least the functions are working now huh.
